sw/device/lib/dif/dif_rstmgr.c

```diff
@@ -139,37 +142,31 @@ dif_rstmgr_result_t dif_rstmgr_reset_is_locked(
   }
 
   mmio_region_t base_addr = handle->params.base_addr;
-  *is_locked = rstmgr_software_reset_locked(base_addr, peripheral);
+  *is_locked = rstmgr_software_reset_is_locked(base_addr, peripheral);
 
   return kDifRstmgrOk;
 }
 
-dif_rstmgr_result_t dif_rstmgr_causes_get(
-    const dif_rstmgr_t *handle, dif_rstmgr_causes_bitfield_t *causes) {
-  if (handle == NULL || causes == NULL) {
+dif_rstmgr_result_t dif_rstmgr_reset_info_get(
+    const dif_rstmgr_t *handle, dif_rstmgr_reset_info_bitfield_t *info) {
+  if (handle == NULL || info == NULL) {
     return kDifRstmgrBadArg;
   }
 
   mmio_region_t base_addr = handle->params.base_addr;
-  uint32_t reg = mmio_region_read32(base_addr, RSTMGR_RESET_INFO_REG_OFFSET);
-
-  *causes = bitfield_field32_read(reg, kRstmgrCausesField);
+  *info = mmio_region_read32(base_addr, RSTMGR_RESET_INFO_REG_OFFSET);
 
   return kDifRstmgrOk;
 }
 
-dif_rstmgr_result_t dif_rstmgr_causes_clear(const dif_rstmgr_t *handle) {
+dif_rstmgr_result_t dif_rstmgr_reset_info_clear(const dif_rstmgr_t *handle) {
   if (handle == NULL) {
     return kDifRstmgrBadArg;
   }
 
   mmio_region_t base_addr = handle->params.base_addr;
```

Comment:
nit: can inline this.

Reply:
I prefer it this way, to reduce splitting long lines (in some `mmio_region_write32` where this is used).